saged/metadata_to_tsv.R

```diff
@@ -0,0 +1,25 @@
+# This script writes the metadata generated by download_recount3.R to a tsv
```

Comment:
I'm confused. I'm guessing you're doing this because the metadata is big enough that it's unwieldy to write it out as a tsv in `download_recount3.R`, but if it's that big, why are you writing it out as an R object bunch of times, wouldn't that be slow/unwieldy too?

Reply:
I'm trying to make the data downloading code as independent of the rest of my pipeline as possible. Afaict no one has put together a script that downloads all of recount3 yet, so I will hopefully give the code to the recount3 people or publish it somewhere. If I do, I don't want future scientists to be bound to tsvs just because that's what I was using in my pipeline.

All of which is to say that you're correct, it would make more sense to write out the tsv in `download_recount3`, but I don't currently plan on doing so